modules/csharp/examples/code_snippets/Program.cs

```diff
@@ -449,6 +449,15 @@ private static void CloseDatabase()
             // end::close-database[]
         }
 
+        private static void DatabaseFullsync()
+        {
+           // tag::database-fullsync[] 
+          var config = new DatabaseConfiguration();
```

Comment:
This part is probably redundant.  You can save some space by not including it in the tag.

Reply:
Removed.